### PR TITLE
Account Management

### PR DESCRIPTION
**Pull request by hellen-22**
# Description

Allow viewing, updating and deleting of user accounts

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

[CI Django & Postgres Tests](https://github.com/SpaceyaTech/blog/actions/workflows/django-postgres-ci.yml)

[Jambo](https://github.com/SpaceyaTech/mastori/actions/workflows/jambo.yaml)

**Test Configuration**:

```sql
          python manage.py migrate
          python manage.py test
```

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by aibunny — DISMISSED** [review]

**Review by JimmyTron — DISMISSED** [review]

**Review by Saint-Chris-7 — DISMISSED** [review]

File: accounts/views.py [hunk 1]
```diff
@@ -58,3 +58,14 @@ class RegisterAccountViewSet(CreateModelMixin, GenericViewSet):
     throttle_classes = [UserRateThrottle, AccountsRateThrottle]
 
 
+class AccountViewSet(ModelViewSet):
+    queryset = Account.objects.all()
+    http_method_names = ['get', 'put', 'patch', 'delete']
+    throttle_classes = [UserRateThrottle, AccountsRateThrottle]
+    #permission_classes = [IsAuthenticated]
```
Comment (JimmyTron) [inline accounts/views.py:65]: Looks like you forgot to uncomment the permision class.
I could view accounts even without loging in
Screenshot from 2023-03-30 03-28-46

File: accounts/serializers.py
Comment (JimmyTron) [inline accounts/serializers.py:1]: Is it feasible to probably have less serializer or having several. I see some serializers having simmilar data. is it possible to refactor

**Review by JimmyTron — CHANGES_REQUESTED** [review]
Check on the auth of the endpoints

**Review by JimmyTron — APPROVED** [review]

File: accounts/serializers.py
Comment (hellen-22) [reply accounts/serializers.py:1]: Yes, I'll refactor

**Review by Saint-Chris-7 — APPROVED** [review]

File: accounts/views.py (hunk 1 above)
Comment (JimmyTron) [reply accounts/views.py:65]: ok

**Review by JimmyTron — APPROVED** [review]

**Review by Saint-Chris-7 — APPROVED** [review]

**Review by aibunny — APPROVED** [review]

**Pull request closed without merging**